File: fizzbuzz/fizzbuzz.dart
void main() {
  for (var i = 1; i <= 100; i++) {
    var output = "";
    if (i % 3 == 0) {
      output += "Fizz";
    }
    if (i % 5 == 0) {
      output += "Buzz";
    }
    if (output.isEmpty) {
      output = i.toString();
    }
    print(output);
  }
}

// To run:
// dart fizzbuzz.dart

File: fizzbuzz/fizzbuzz.js
for (let i = 1; i <= 100; i++) {
  let output = "";

  if (i % 3 === 0) {
    output += "fizz";
  }

  if (i % 5 === 0) {
    output += "buzz";
  }

  console.log(output || i); // Output "fizz", "buzz", "fizzbuzz", or the number itself
}

// How to run:
// node fizzbuzz.js
